website/static/js/components/mini-repl.js

```diff
@@ -1,13 +1,21 @@
 /* globals Babel */
 
-import debounce from "https://cdn.skypack.dev/lodash-es/debounce";
+import debounce from "https://cdn.skypack.dev/pin/lodash-es@v4.17.21-rDGl8YjBUjcrrAbjNrmo/mode=imports,min/unoptimized/debounce.js";
```

Comment:
Pin `loadsh-debounce` to an optimized minified specific option to prevent breaking changes.